examples/hosts/app-integration/external-views/package.json

```diff
@@ -35,7 +35,12 @@
   "dependencies": {
     "@fluidframework/aqueduct": "^0.31.0",
     "@fluidframework/get-session-storage-container": "^0.31.0",
-    "@fluidframework/get-tinylicious-container": "^0.31.0"
+    "@fluidframework/get-tinylicious-container": "^0.31.0",
```

Comment:
All of these changes are in the /external-views sample -- you should instead modify the /external-controller sample

Reply:
Yikes... I missed this. Thanks @ChumpChief.

Reply:
haha! wrong folder :D